sdk/spring-sdk/src/main/java/kalix/springsdk/annotations/Entity.java

```diff
@@ -16,13 +16,12 @@
 
 package kalix.springsdk.annotations;
 
-import java.lang.annotation.ElementType;
-import java.lang.annotation.Retention;
-import java.lang.annotation.RetentionPolicy;
-import java.lang.annotation.Target;
+import java.lang.annotation.*;
 
-@Target({ElementType.TYPE})
+@Target(ElementType.TYPE)
 @Retention(RetentionPolicy.RUNTIME)
-public @interface EntityKey {
-  String[] value() default {};
+@Documented
+public @interface Entity {
+  String entityType();
+  String[] entityKey();
```

Comment:
@johanandren, I replaced EntityKey by this one. 

Defining an Entity requires the entityKey and entityType, always. So easier if user needs to deal with only one annotation. And both fields are mandatory. 

Also, as discussed here: https://github.com/lightbend/kalix-jvm-sdk/issues/1016
I don't think we should allow entityKeys in different places.